Dockerfile

```diff
@@ -48,4 +48,5 @@ RUN chmod ug+rwx /usr/share/nginx/html/
 EXPOSE 8080
 USER 101
 # Start Nginx server
-CMD ["/bin/sh", "-c", "/usr/share/nginx/html/env.sh && nginx -g \"daemon off;\""]
+CMD ["nginx", "-g", "daemon off;"]
+#CMD ["/bin/bash", "-c", "/usr/share/nginx/html/env.sh && nginx -g \"daemon off;\""]
```

Comment:
better to remove commented line, anyway you can always look for it in git history

Reply:
@adityagajbhiye9  pls check.

Reply:
I have removed in PR #45 .

Reply:
@dvasunin  Please review.

Reply:
already approved